src/redux/user/validation/sign-in-schema.ts

```diff
@@ -0,0 +1,13 @@
+import * as Yup from "yup";
+
+import { emailRegex, userValidationMessage } from "./constants/index.ts";
+
+const userSignInValidation = Yup.object().shape({
```

Comment:
why its in redux? not near the component where you use this validation

Reply:
I've changed it